modules/supporting_encapsulation.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/optimizing-cpu-usage.adoc
+
+[id="supporting-encapsulation_{context}"]
+= Supporting encapsulation
+
+As an OpenShift support engineer or developer, you want to inspect Kubernetes-specific mountpoints.
+
+== Locating the encapsulated namespaces
+
+Any monitoring tool that relies on the ability to run in the host OS and have visibility of mountpoints created by Kubelet, the Container Runtime, or containers themselves, will need to enter the container mount namespace in order to see these mountpoints. The provided `kubensenter` script will execute another command inside the Kubernetes mountpoint and can be used to adapt any existing tools.
+The `kubensenter` script is aware of the state of the mount encapsulation feature status, and is safe to run even if encapsulation is not enabled. In that case it will simply execute the provided command in the default mount namespace.
+For example, if a systemd service needs to run inside the new Kubernetes mount namespace, you should edit the service file and prefix the `ExecStart=` command line with `kubensenter`.
+
+[source,terminal]
+----
+[Unit]
+Description=Example service
+[Service]
+ExecStart=/usr/bin/kubensenter /path/to/original/command arg1 arg2
+----
+
+
+== Inspecting the encapsulated namespaces
+
+An administrator logged in to the host OS wishing to obtain information about the Kubernetes system for debugging or auditing purposes will need to be aware of the new mount namespace. A shell originating in a container, such as `oc debug` will already be inside the Kubernetes namespace. A shell originating in an SSH session will be in the default namespace, and an administrator will need to run the `kubensenter` script as root to view or interact with these mountpoints.
+The `kubensenter` script is aware of the state of the mount encapsulation feature status, and is safe to run even if encapsulation is not enabled. In that case it will simply execute the shell or requested command in the default mount namespace.
+
+
+To run a single command inside the Kubernetes namespace, provide the command and any arguments to the `kubensenter` script.  For example, to run the `findmnt` command inside the Kubernetes namespace, issue the following command:
+[source,terminal]
+----
+$ sudo kubensenter findmnt
```

Comment:
Where are we running this from? inside the cluster host? I think it would be good to add steps before this command to clearly outline where you run this command from. ie, something like: 

1. Open a remote shell to the cluster host or `oc debug node/<node-name>`

...

Reply:
In this case, it would be more important if the admin had SSHd in instead of `oc debug`, but I don't know if that's something we want to encourage users to do?  It is important for people to know, if they do SSH in, though.

Reply:
Will resolve with Jim.